src/goto-programs/resolve_concrete_function_call.cpp

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************\
+
+ Module: Goto Program utilties
```

Comment:
Not sure exactly what this comment wants the name to change to? I've corrected the capitalisation if that's what you meant?